x-pack/plugin/async-search/src/test/java/org/elasticsearch/xpack/search/AsyncSearchActionIT.java

```diff
@@ -277,4 +279,117 @@ public void testCancellation() throws Exception {
         deleteAsyncSearch(response.getId());
         ensureTaskRemoval(response.getId());
     }
+
```

Comment:
perhaps you can now remove this TODO? https://github.com/elastic/elasticsearch/commit/146b2a85b42220a5f653a543a5107d8f960464d5#diff-97b9cc86781ba2021cd3ae58d6851834R38

Reply:
That's already removed in this change:
https://github.com/elastic/elasticsearch/pull/55435/files#diff-dcd3d2ba4a8ca88c8655de7eb18981e6L38

Reply:
great I did not notice